### PR TITLE
patchelf the coda-libp2p_helper

### PR DESCRIPTION
**Pull request by emberian**
Without this, INTERP is set to a dangling path and you'll see `bash: /usr/local/bin/coda-libp2p_helper: No such file or directory` when trying to run the helper in a container.

### CODE REVIEW

File: scripts/rebuild-deb.sh [hunk 1]
```diff
@@ -58,7 +58,7 @@ echo "------------------------------------------------------------"
 mkdir -p "${BUILDDIR}/usr/local/bin"
 cp ./default/src/app/cli/src/coda.exe "${BUILDDIR}/usr/local/bin/coda"
 cp "${SCRIPTPATH}/../src/app/libp2p_helper/result/bin/libp2p_helper" "${BUILDDIR}/usr/local/bin/coda-libp2p_helper"
-patchelf --set-interpreter /lib64/ld-linux-x86-64.so.2 "${BUILDDIR}/usr/local/bin/coda-libp2p_helper"
+sudo patchelf --set-interpreter /lib64/ld-linux-x86-64.so.2
```
Comment (psteckler) [inline scripts/rebuild-deb.sh]: don't you still need the libp2p helper file argument?

Comment (yourbuddyconner) [reply scripts/rebuild-deb.sh]: @psteckler Can you give an example argument, I have less context with what this is doing

Comment (psteckler) [reply scripts/rebuild-deb.sh]: @yourbuddyconner I thought the required change was to add `sudo`, but the actual file for the interpreter was omitted in the command (the `$BUILDDIR)/usr....` file).

Comment (bkase) [reply scripts/rebuild-deb.sh]: I don't think we can use `sudo` during a build. Is it demanded here?

Comment (emberian) [reply scripts/rebuild-deb.sh]: good catch, yes.

Comment (emberian) [reply scripts/rebuild-deb.sh]: @bkase I think it was a dangling symlink error, not a permission error, so no.

**Pull request merged**